torch/fx/experimental/symbolic_shapes.py

```diff
@@ -243,6 +255,28 @@ def bool_(self):
 
 
 if HAS_SYMPY:
+    # Overloaded to be compatible with regular Python.
+    # https://github.com/pytorch/pytorch/issues/90900
+    class Pow(sympy.Function):
+        @classmethod
+        def eval(cls, base, exp):
+            if exp.is_zero:
+                return sympy.Integer(1)
+            elif base.is_zero and exp < 0:
+                raise ZeroDivisionError(f"{base} cannot be raised to a negative power")
+            else:
+                return base ** exp
+
+    # Overloaded to be compatible with regular Python.
+    # https://github.com/pytorch/pytorch/issues/90900
+    class TrueDiv(sympy.Function):
+        @classmethod
+        def eval(cls, base, divisor):
+            if divisor.is_zero:
+                raise ZeroDivisionError("division by zero")
+            else:
+                return base / divisor
```

Comment:
Probably need to manually implement the assumptions stuff here too

Reply:
I don't think so. These never return None, so they are replaced with the underlying sympy exprs, eg:

```
[ins] In [3]: Pow(sympy.Symbol("y"),sympy.Symbol("x"))
Out[3]: y**x
```